### ISSUE TITLE
Add an option to scan only a specific modulation or symbol rate

### ISSUE THREAD

**Issue by oCanna** [stefantalpalaru/w_scan2#11]
Hello,

it would be greate if there was an option to specify the symbol rate and/or modulation for DVB-S/S2 scans to speed up the scan process a little bit. Same as it is already done for DVB-C.

Kind regards,
Bart

**Comment by Marco-GG** [comment]
Maybe I'm wrong (I don't have hard to test Sat beams... yet :P), but DVB-S/S2 scans seems to be executed by selecting the satellite to scan from a hard-coded list in w_scan2. This is: there is a list of satellites stored on "satellites.dat" and you can setup w_scan2 to search channels on one of them.

So, each entry of satellites.dat is a vector of elements of type __sat_transponder (i.e. each element is a transponder for a satellite), which has the information of the symbol rate and modulation for the transponders.

Long story short: maybe what you're asking is to allow the dynamic parsing of transponders information?

**Comment by oCanna** [comment]
Didn't know that there is a list of sattelites/transponders and now I see you can even specify your own list (initial tuning data) of transponders. That is all I need. Sorry I bothered you.
